tasks/images.js

```diff
@@ -21,9 +21,9 @@ gulp.task('images', callback => {
     runSequence(
         'clean:images',
         ['copy:img', 'copy:assets'],
-        'images:optimise',
+        // 'images:optimise',
```

Comment:
Rather than remove these, I'd make them able to be configured off by config params (similar to the docs copy task on line 25). Just to make sure this isn't a breaking change on older platforms